extensions/src/ACRE2Mumble/MumbleClient.cpp

```diff
@@ -324,6 +325,7 @@ uint64_t CMumbleClient::findChannelByNames(std::vector<std::string> details_) {
             char *channelName = nullptr;
 
             if (mumAPI.getChannelName(pluginID, activeConnection, channelId, &channelName) == STATUS_OK) {
+				// TODO: Does std::string take care of deleting the allocated name?
                 std::string channelNameString(channelName);
```

Comment:
:point_up:

Reply:
Yes, its destructor will deallocate it.

Reply:
@jonpas are you sure? From what I see at https://www.cplusplus.com/reference/string/string/string/ it seems that this constructor only copies the given C-string.
In general I think std::string can't even properly discard this String as it needs to be done via Mumble's API anyway... I think that's a bug :eyes:

Reply:
I changed the implementation to what _I_ think is correct at this point. If that is not what is wanted/needed here, we can talk about it again in the following review :)